test/kubemark/start-kubemark.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script that creates a Kubemark cluster with Master running on GCE.
```

Comment:
It actually implements hybrid option (Kubelets/Nodes as Pods, Master on a dedicated VM) for now, as it was easier to do. HollowNode was supposed to be HollowKubelet+HollowProxy, and because we don't have HollowProxy just yet, it's the same as running multiple Kubelet's replicas. Please take a look at the list #13731.